src/app/components/editors/gameobject/gameobject-template/gameobject-template.component.html

```diff
@@ -0,0 +1,104 @@
+          <input [formControlName]="'name'" id="name" type="text" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="type">type</label>
+          <app-single-value-selector-btn
+            [control]="editorService.form.get('type')"
+            [config]="{ options: GAMEOBJECT_TYPE, name: 'type' }"
+            [modalClass]="'modal-md'"
+          ></app-single-value-selector-btn>
+          <input [formControlName]="'type'" id="type" type="number" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="displayId">displayId</label>
+          <i
+            class="fas fa-info-circle ml-1" [placement]="'auto'"
+            [tooltip]="'Graphic model id sent to the client from GameObjectDisplayInfo.dbc'"
+          ></i>
+          <input [formControlName]="'displayId'" id="displayId" type="text" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="IconName">IconName</label>
+          <app-single-value-selector-btn
+            [control]="editorService.form.get('IconName')"
+            [config]="{ options: GAMEOBJECT_ICON, name: 'IconName' }"
+            [modalClass]="'modal-xl'"
+          ></app-single-value-selector-btn>
+          <input [formControlName]="'IconName'" id="IconName" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="size">size</label>
+          <input [formControlName]="'size'" id="size" type="text" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="AIName">AIName</label>
+          <input [formControlName]="'AIName'" id="AIName" type="text" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="ScriptName">ScriptName</label>
+          <input [formControlName]="'ScriptName'" id="ScriptName" type="text" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="castBarCaption">castBarCaption</label>
+          <input [formControlName]="'castBarCaption'" id="castBarCaption" type="text" class="form-control form-control-sm">
+        </div>
+        <div class="form-group col-12 col-sm-6 col-md-4 col-lg-3 col-xl-2">
+          <label class="control-label" for="unk1">unk1</label>
+          <input [formControlName]="'unk1'" id="unk1" type="text" class="form-control form-control-sm">
```

Comment:
remove all `type="text"` since the default type for `<input>` is text so there is no need to specify it